core/src/main/java/org/javarosa/xform/util/XFormUtils.java

```diff
@@ -61,6 +63,39 @@ public static FormDef getFormRaw(InputStreamReader isr) throws XFormParseExcepti
         return _factory.getXFormParser(isr).parse();
     }
 
+    public static FormDef getFormFromInputStream(InputStream is, QuestionExtensionParser extensionParser)
```

Comment:
I'd guess we want this to be a list or a vector to make it easier to add a bunch of these.